### DISCUSSION TITLE
Wrong amount of points for 5 kyu kata? (8 instead of 21)

### DISCUSSION THREAD

**Discussion by Tom-H-L** [codewars/codewars.com#2941] [Q&A]
Hi, 
I am 3. kyu and have just done two 5 kyu katas. I got 8 points for each of them, although in the table [here](https://docs.codewars.com/gamification/ranks/) it says I should have gotten 21 points. Why is that so?

The katas I did were:

1. https://www.codewars.com/kata/529b418d533b76924600085d
2. https://www.codewars.com/kata/59de1e2fe50813a046000124

**Comment by hobovsky** [comment]
Did you read the paragraph "Leveling your rank", the one just above the table you cite?

**Reply by Tom-H-L** [reply]
Thanks for the hint! I was aware of that, but I always assumed that the honor I get is >= the rank points for each kata. So I wrongly assumed that if I see the counter for honor go up only 8 points, then the rank points I got will be even less. But it seems I was wrong in that assumption. Thank you for pointing that out to me!
